packages/flutter_tools/test/general.shard/base/error_handling_io_test.dart

```diff
@@ -48,6 +48,11 @@ void main() {
      expect(ErrorHandlingFileSystem.deleteIfExists(file), true);
   });
 
+  testWithoutContext('create accepts exclusive argument', () {
+    final FileSystem fileSystem = MemoryFileSystem.test();
+    fileSystem.file('file').create(exclusive: true);
```

Comment:
nit: this should have some kind of expectation in the test :)

Reply:
Done!